pallets/loans/src/lib.rs

```diff
@@ -14,6 +14,19 @@
 //! # Loan pallet
 //!
 //! This pallet provides functionality for managing loans on Tinlake
+//!
+//! Check the next table as an overview of the possible actions you can do for managing loans:
+//!
+//! |      Action     |   From  |    To   |      Role     | Collateral Owner |
+//! |-----------------|---------|---------|---------------|------------------|
+//! |      create     |         | Created |    Borrower   |        Yes       |
+//! |      price      | Created |  Active | PricingAdmin |                  |
+//! |      borrow     |  Active |  Active |               |        Yes       |
+//! |      repay      |  Active |  Active |               |        Yes       |
+//! |    write_off    |  Active |  Active |               |                  |
+//! | admin_write_off |  Active |  Active |   LoanAdmin   |                  |
+//! |      close      |  Active |  Closed  |               |        Yes       |
```

Comment:
```suggestion
//! |      create     |         | Created |    Borrower   |        Yes       |
//! |      price      | Created |  Active | PricingAdmin  |                  |
//! |      borrow     |  Active |  Active |               |        Yes       |
//! |      repay      |  Active |  Active |               |        Yes       |
//! |    write_off    |  Active |  Active |               |                  |
//! | admin_write_off |  Active |  Active |   LoanAdmin   |                  |
//! |      close      |  Active |  Closed |               |        Yes       |
```

Reply:
Looks like I broke the indentation

Reply:
I've seen it later, I'll fix it in any of the following PRs. No worries 👍🏻